core-implementation/search_transaction_by_user/search_transaction_by_user.ts

```diff
@@ -0,0 +1,36 @@
+/*
+
+Source for comments: https://chatgpt.com/share/67213342-70c0-8003-91ef-8b6c29b25934
+
+*/
+
+
+import { handleUnaryCall } from "@grpc/grpc-js";
+import { core } from "../../generated/core/core.js";
+
+// Populated with transaction data - This acts as an in-memory database for transaction records.
+export const ledger: core.Transaction[] = [];
```

Comment:
just want to make sure you know: you didnt need to do this for sprint 1, you just needed to return hardcoded data.

Reply:
(no need to change anything here)